rs-e2e/src/main/java/gov/hhs/cdc/trustedintermediary/rse2e/AzureBlobFileFetcher.java

```diff
@@ -31,8 +35,11 @@ private AzureBlobFileFetcher() {
         this.blobContainerClient =
                 new BlobContainerClientBuilder()
                         .connectionString(azureStorageConnectionString)
-                        .containerName(azureStorageConnectionName)
+                        .containerName(CONTAINER_NAME)
                         .buildClient();
+
+        AzureBlobOrganizer blobOrganizer = new AzureBlobOrganizer(blobContainerClient);
+        blobOrganizer.organizeAndCleanupBlobsByDate(RETENTION_DAYS, TIME_ZONE);
```

Comment:
Hmmm that's a good point. That reminds me why we want to change from UTC to another timezone. I have another story where I'll change the timezone to avoid this kind of issues. That also reminds me I should I add something in the readme about the retention time for the files